build.gradle.kts

```diff
@@ -15,6 +15,7 @@ buildscript {
         classpath(libs.com.android.tools.build.gradle)
         classpath(libs.org.jetbrains.kotlin.serialization)
         classpath(libs.com.squareup.sqldelight.gradle.plugin)
+        classpath(libs.com.google.osdetector.gradle.plugin)
```

Comment:
I've commented why there was an issue with applying plugin with version via version catalog, but this one is fine as well 👍

Reply:
I learned something thanks to your comment 🙏